kafka-ui-react-app/src/components/Topics/Topic/Messages/Message.tsx

```diff
@@ -116,16 +106,20 @@ const Message: React.FC<Props> = ({
         <td>
           <div>{formatTimestamp(timestamp)}</div>
         </td>
-        <StyledDataCell title={key}>
-          {renderFilteredJson(key, keyFilters)}
-        </StyledDataCell>
-        <StyledDataCell title={content}>
+        <S.StyledDataCell title={key}>
+          <Ellipsis serde={keySerde}>
+            {renderFilteredJson(key, keyFilters)}
+          </Ellipsis>
+        </S.StyledDataCell>
+        <S.StyledDataCell title={content}>
           <S.Metadata>
             <S.MetadataValue>
-              {renderFilteredJson(content, contentFilters)}
+              <Ellipsis serde={valueSerde}>
```

Comment:
Ellipsis is a common component. why we sending serde property here?

Reply:
done